data/language/en-GB.txt

```diff
@@ -230,7 +230,7 @@ STR_0825    :Chosen name in use already
 STR_0826    :Too many names defined
 STR_0827    :Not enough cash - requires {CURRENCY2DP}
 STR_0828    :{SMALLFONT}{BLACK}Close window
-STR_0829    :{SMALLFONT}{BLACK}Window title - Drag this to move window
+STR_0829    :{SMALLFONT}{BLACK}Window title - drag this to move window
```

Comment:
This is an instance where the capital serves to show two separate thoughts that are simply not using standard punctuation.  Namely, the full thought is "This is the window title.  If you drag this it will move the window."  We're just condensing the information down to the minimum readable size.  I would say that either leave it as it was or change the - punctuation to something that makes the split more obvious.